crates/worker/src/partition/state_machine/command_interpreter.rs

```diff
@@ -569,27 +608,38 @@ where
         invocation_metadata: InvocationMetadata,
         error: InvocationError,
     ) -> Result<(), Error> {
+        let length = invocation_metadata.journal_metadata.length;
+
+        self.try_send_failure_response(effects, &full_invocation_id, invocation_metadata, error);
+        self.complete_invocation(full_invocation_id, state, length, effects)
```

Comment:
This method name is very misleading. Something like `end_invocation_lifecycle`, or somewhat like that, might make more obvious what it does.

Reply:
Will rename it.